packages/bpk-scrim-utils/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "bpk-scrim-utils",
+  "version": "1.0.0",
+  "description": "Higher order component that adds an overlay behind components and manages scroll states.",
```

Comment:
This description seems overly focused on the one HOC, whereas the pakcage name is called suffixed `-utils` implying plural. Check out https://www.npmjs.com/package/bpk-react-utils (not that it's a good example, but it deals with the fact that there are multiple).